### DISCUSSION TITLE
Czy soft nadaje sie do Radia Quansheng UV-5R PLUS ?

### DISCUSSION THREAD

**Discussion by DajMiliona** [egzumer/uv-k5-firmware-custom#233] [Q&A]
Na FB polecali Pana soft. Czy działa on tez z tym modelem tj. Quansheng UV-5R PLUS?

**Comment by ErikS-web** [comment]
Yes it does.... 
See #109 and #135

p.s.
your battery-capacity is propably 2200 and not 1600. 
See Menu-item 68. `BatTyp`

**Reply by DajMiliona** [reply]
It is posiible to disable TX from menu radio?

**Reply by ErikS-web** [reply]
Read [the manual, under hidden menu](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Menu#hidden-menu) 
Menu 61 `F Lock`; DISABLE ALL - **disables TX on all frequencies**
